bool Claw_done
bool Dropper_done
bool Torpedo_done
